helm/cmsweb/dbs2go/templates/deployment.yaml

```diff
@@ -0,0 +1,141 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "dbs2go.fullname" . }}
+  labels:
+    {{- include "dbs2go.labels" . | nindent 4 }}
+  namespace: dbs
+spec:
+  {{- if not .Values.autoscaling.enabled }}
+  {{- if or .Values.environment.prod .Values.environment.preprod}}
+  replicas: {{ .Values.replicaCount.prod }}
+  {{- else }}
+  replicas: {{ .Values.replicaCount.test }}
+  {{- end }}
+  {{- end }}
+  selector:
+    matchLabels:
+      {{- include "dbs2go.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "dbs2go.selectorLabels" . | nindent 8 }}
+        {{- if .Values.environment.preprod}}
+        env: k8s-preprod
+        {{- end }}
+        {{- if .Values.environment.prod}}
+        env: k8s-prod
+        {{- end }}
+        {{- if .Values.environment.test}}
+        env: k8s
+        {{- end }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ include "dbs2go.serviceAccountName" . }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      containers:
+        - name: {{ .Chart.Name }}
+          securityContext:
```

Comment:
There is no securitycontext in dbs2go file https://github.com/dmwm/CMSKubernetes/blob/master/kubernetes/cmsweb/services/dbs2go.yaml

Reply:
`securityContext` is still there in deployment.yaml. Please remove this